modules/apps/frontend-js/frontend-js-test/src/testFunctional/A11yTool.path

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+<title>A11yTool</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">A11yTool</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>ACCESSIBILITY_PANEL_HEADING</td>
+	<td>//span[contains(text(),'Accessibility Violations')]</td>
```

Comment:
I'd recommend using semantic tags if possible and for the path locator elements/attributes to relate to the target we're interested in. ie: the name of the path is ACCESSIBILITY_PANEL_HEADING so I'd search for key attribute names such as class="sidebar" or "sidenav" or "sidebar-header" to include in path locator.

Some other guidelines I recommend:
https://docs.google.com/spreadsheets/d/1KgMk5CoAB0Kqylg2dufw9X4zGenrWNEv13KdVckrkvg/edit#gid=1536975616

example: Assert ADD_PANEL_HEADER `//*[contains(@class,'lfr-add-panel')]//*[contains(@class,'sidebar-header')]`
![screenshot_7](https://user-images.githubusercontent.com/35235266/137970426-943a9f99-85f8-45fb-80c3-6bed5ced644e.png)

This is nit picky but crafting a robust locator is something I like to invest in.